services/slam/server_test.go

```diff
@@ -171,58 +379,52 @@ func TestServer(t *testing.T) {
 	t.Run("failing on improper service interface", func(t *testing.T) {
 		improperImplErr := slam.NewUnimplementedInterfaceError("string")
 
+		// Get position
```

Comment:
Thanks for adding these sections. Doesn't need to change in this PR but b/c each of these calls is independent of the others IMO, they should each be in a different t.Run.

Reply:
kk made a ticket for this